tests/fixtures/fixture_preprocess.py

```diff
@@ -0,0 +1,60 @@
+import pytest
```

Comment:
I'd never say writing tests is a bad thing, but I would suggest that given you don't think this method works tremendously well, to perhaps consider only writing tests once you think that a particular approach is bearing fruit. Tests take a long time to write and maintain and adding them for an experiment might not be a good use of time. Just a suggestion though 😄

Reply:
Thanks, actually I write the tests and fixtures before actually trying to see if my function works on the actual data as they are helpful in allowing me to get a minimal reproducible example. Sometimes the actual data such as the preprocessed content store can be quite large and have a lot of noise relative to what I'm trying to achieve so writing my fixtures help me think of ways I can simplify the problem. In no ways are my tests comprehensive nor do they consider all the possible ways things could go wrong (which touches on your point). 🙂